tools/toolchains/gcc.py

```diff
@@ -219,6 +221,13 @@ def link(self, output, objects, libraries, lib_dirs, mem_map):
         # Build linker command
         map_file = splitext(output)[0] + ".map"
         cmd = self.ld + ["-o", output, "-Wl,-Map=%s" % map_file] + objects + ["-Wl,--start-group"] + libs + ["-Wl,--end-group"]
+        # Create Secure library
+        if self.target.core == "Cortex-M23" or self.target.core == "Cortex-M33":
+            secure_file = "cmse_lib.o"
+            secure_file = join(split(output)[0], secure_file)
```

Comment:
Please use 
```python
secure_file = join(dirname(output), "cmse_lib.o")
```
instead.

Reply:
Team: cmse_lib.o : Is it ok to name secure library as generic secure lib, or should it be core specific like CortexM23-S-lib.o?

Reply:
For reference, if you are going to name it that way, the python should look like:
```python
secure_file = join(dirname(output), "%s-S-lib.o" % self.target.core)
```